self.app.opt.sound is a boolean value while the 
self.tkopt.sound.set() function expects a 0 or 1.

Small hack since i'm no python ninja.

http://bugs.gentoo.org/94234

--- src/actions.py
+++ src/actions.py
@@ -753,7 +753,11 @@
     def mOptSoundDialog(self, *args):
         if self._cancelDrag(): return
         d = SoundOptionsDialog(self.top, "Sound settings", self.app)
-        self.tkopt.sound.set(self.app.opt.sound)
+        if self.app.opt.sound:
+            setit = 1
+        else:
+            setit = 0
+        self.tkopt.sound.set(setit)
 
     def mOptAnimations(self, *args):
         if self._cancelDrag(): return
